### ISSUE TITLE
Google Analytics is used to track users (critical privacy issue)

### ISSUE THREAD

**Issue by Symbian9** [MoonchildProductions/Pale-Moon#1237]
* https://github.com/mozilla/addons-frontend/issues/2785
* https://github.com/RequestPolicy/requestpolicy/issues/473

Affected:
* Mozilla Firefox
  * Pale Moon
  * TOR Browser
  * other Firefox-based forks

**Comment by mattatobin** [comment]
Our Discover Pane (Get Add-ons) calls a page from our own servers NOT from AMO. This should be obvious because it isn't the same page.

See:
http://xref.palemoon.org/palemoon-trunk/source/browser/branding/shared/pref/preferences.inc#4
http://xref.palemoon.org/palemoon-trunk/source/browser/branding/shared/pref/preferences.inc#100
https://github.com/Pale-Moon-Addons-Team/phoebus/blob/RELEASE/phoebus/components/discover/discover.php
https://github.com/Pale-Moon-Addons-Team/phoebus/blob/RELEASE/phoebus/components/discover/content/template.xhtml
https://github.com/Pale-Moon-Addons-Team/phoebus/blob/RELEASE/phoebus/components/discover/content/content.xhtml

AND THE FINAL RESULTING PAGE - https://addons.palemoon.org/?component=discover

tl;dr The Discover Pane does not track you and there is no facility for doing so on the server side and the sending all your extensions and state via a huge json request is disable on the browser side and is actually in almost all cases too long on the server side and would be rejected anyway.

We have been using our own page for that since Pale Moon 25.

Thus this is RESOLVED INVALID.

**Comment by Symbian9** [comment]
> We have been using our own page for that since Pale Moon 25.

Issue description fixed

**Comment by mattatobin** [comment]
Your description could use quite a bit of work.. I had to read your other rant-y bugs elsewhere to figure out exactly what you were talking about.

So tell me, did you just do a general web search for browsers based in part or in full on Firefox to spread your discontent? You do realize that you do not have to be told every little thing about what data people collect on you right only that it is possible and your use of a product and service implies consent for it.

In the future I would expect you to verify issues with Pale Moon specifically before posting an issue.. Maybe ask on the forums or the IRC channel before creating such a bug report?

**Comment by wolfbeast** [comment]
Pale Moon is NOT affected. Why are you saying it is? Did you even check?
